features/conference/components/Conference.js

```diff
@@ -18,13 +27,25 @@ class Conference extends Component {
      */
     render() {
         return (
-            <ConferenceContainer>
+            <ConferenceContainer onPress = { this.onPress.bind(this) }>
                 <LargeVideo/>
-                <Toolbar navigator = { this.props.navigator }/>
-                <FilmStrip/>
+                <Toolbar
```

Comment:
Well, you have ```isToolbarVisible``` value inside Conference's state, so it won't be lost.
As for performance issue, you might be probably right for Filmstrip, especially with current bug that RTCView doesn't release memory when removed.
But having many video streams mounted and playing might also affect performance and battery life. So if toolbar buttons are something that is shown by default/more often, I'd go with removing elements. But if FilmStrip is something that is shown by default/more often,
then hiding seems to be better solution, at least until memory leaks with RTCView are fixed.